apps/language_server/test/experimental/code_mod/format_test.exs

```diff
@@ -71,6 +71,76 @@ defmodule ElixirLS.Experimental.FormatTest do
       ]t == result
     end
 
+    test "it can split a long line to two lines" do
+      unformatted = ~q[
+        defmodule Unformatted do
+          def foo1(s) do
+            s = very_loooooooooooooooooooooooooooooooooooooooooooooooooooooooooooonooooong(s) |> IO.inputs()
+            s
+          end
+
+          def very_loooooooooooooooooooooooooooooooooooooooooooooooooooooooooooonooooong(s) do
+            s
+          end
+        end
+      ]t
+
+      formatted = ~q[
+        defmodule Unformatted do
+          def foo1(s) do
+            s =
+              very_loooooooooooooooooooooooooooooooooooooooooooooooooooooooooooonooooong(s) |> IO.inputs()
+
+            s
+          end
+
+          def very_loooooooooooooooooooooooooooooooooooooooooooooooooooooooooooonooooong(s) do
+            s
+          end
+        end
+      ]t
+
+      assert {:ok, formatted} == modify(unformatted)
+    end
+
+    test "it can split a long line into multiple lines" do
+      unformatted = ~q[
+        defmodule Unformatted do
+          def foo(foo1, foo2, foo3, foo4, foo5, foo6, foo7, foo8, foo9, foo10, foo11, foo12, foo13, foo14, foo15, foo16) do
+            foo = foo14 <> foo15 <> foo16
+            {foo1, foo2, foo3, foo4, foo5, foo6, foo7, foo8, foo9, foo10, foo11, foo12, foo13, foo}
+          end
+        end
+      ]t
+      formatted = ~q[
+        defmodule Unformatted do
+          def foo(
+                foo1,
+                foo2,
+                foo3,
+                foo4,
+                foo5,
+                foo6,
+                foo7,
+                foo8,
+                foo9,
+                foo10,
+                foo11,
+                foo12,
+                foo13,
+                foo14,
+                foo15,
+                foo16
+              ) do
+            foo = foo14 <> foo15 <> foo16
+            {foo1, foo2, foo3, foo4, foo5, foo6, foo7, foo8, foo9, foo10, foo11, foo12, foo13, foo}
+          end
+        end
+      ]t
+
+      assert {:ok, formatted} == modify(unformatted)
+    end
```

Comment:
same comment, this looks like two separate tests, can they be split up?

Reply:
There are no two tests here, only the long line function definition with multiple args. I've changed this test for clarity.